README.md

```diff
@@ -1,43 +1,63 @@
 <h1 align="center">
   Awesome Fabric
-  <p align="center"><img src="https://awesome.re/badge-flat.svg" href="https://awesome.re" /></p>
+  <p align="center">
+    <img src="https://awesome.re/badge-flat.svg" href="https://awesome.re" />
+    <img src="https://flat.badgen.net/badge/license/CC0/blue" href="https://creativecommons.org/publicdomain/zero/1.0/" />
+  </p>
 </h1>
 
 <p align="center"><i>A curated list of awesome Fabric resources, libraries and tools - WIP and accepting contributions!</i></p>
 <p align="center"><b><a href="https://fabricmc.net/">Fabric</a> is a mod loader for the voxel game <a href="https://en.wikipedia.org/wiki/Minecraft">Minecraft</a>.</b></p>
 
 ---
 
-<h2 align="center">Contents</h2>
+<h2 align="center">Contents<br /><code>📑</code></h2>
```

Comment:
While the icons look good on the other headings, I don't particularly like it on the "Contents" heading.

Reply:
I have changed it to "🔠", which seems much more logical (at least to me).